src/settings-ui/Settings.UI/SettingsXAML/App.xaml.cs

```diff
@@ -208,17 +209,17 @@ protected override void OnLaunched(Microsoft.UI.Xaml.LaunchActivatedEventArgs ar
             }
             else
             {
+#if DEBUG
                 // For debugging purposes
                 // Window is also needed to show MessageDialog
                 settingsWindow = new MainWindow(isDark);
                 settingsWindow.ExtendsContentIntoTitleBar = true;
                 settingsWindow.Activate();
                 settingsWindow.NavigateToSection(StartupPage);
-
-#if !DEBUG
-                ShowMessageDialogAndExit("The application cannot be run as a standalone process. Please start the application through the runner.", "Forbidden");
+                ShowMessageDialog("The application is running in Debug mode.", "DEBUG");
 #else
-                ShowMessageDialog("The application cannot be run as a standalone process. Please start the application through the runner.", "Forbidden");
+                SettingsDeepLink.OpenSettings(SettingsDeepLink.SettingsWindow.Overview, true);
```

Comment:
```suggestion
                /* If we try to run Settings as a standalone app, it will start PowerToys.exe if not running and open Settings again through it in the General page. */
                SettingsDeepLink.OpenSettings(SettingsDeepLink.SettingsWindow.Overview, true);
```